core/transport/src/main/proto/proto/journal/file.proto

```diff
@@ -284,3 +284,25 @@ message UpdateUfsModeEntry {
   optional string ufsPath = 1;
   optional UfsMode ufsMode = 2 [default = READ_WRITE];
 }
+
+enum CommandType {
+  UNKNOWN = 1;
```

Comment:
Remove `UNKNOWN`.  Why do we want to give people the opportunity to define an unknown command.